iotdb-core/datanode/src/main/java/org/apache/iotdb/db/queryengine/common/schematree/ClusterSchemaTree.java

```diff
@@ -44,23 +46,31 @@
 import java.util.ArrayDeque;
 import java.util.ArrayList;
 import java.util.Deque;
+import java.util.HashMap;
 import java.util.List;
 import java.util.Map;
 import java.util.Set;
 
 import static org.apache.iotdb.commons.conf.IoTDBConstant.PATH_ROOT;
 import static org.apache.iotdb.commons.schema.SchemaConstant.ALL_MATCH_PATTERN;
+import static org.apache.iotdb.commons.schema.SchemaConstant.NON_TEMPLATE;
 import static org.apache.iotdb.db.queryengine.common.schematree.node.SchemaNode.SCHEMA_ENTITY_NODE;
 import static org.apache.iotdb.db.queryengine.common.schematree.node.SchemaNode.SCHEMA_MEASUREMENT_NODE;
 
 public class ClusterSchemaTree implements ISchemaTree {
+  private static final ClusterTemplateManager templateManager =
+      ClusterTemplateManager.getInstance();
```

Comment:
Pass this into the ```deserialize``` method as a local param rather than a global static attribute.

Reply:
> Pass this into the `deserialize` method as a local param rather than a global static attribute.

seems that static attribute can save some memory usgae?

Reply:
The build in static attribute may not friendly to unit test. Anyway, related modification can be delayed to when it is needed.